### ISSUE TITLE
KeyError when importing model with System/International Fonts (Spanish) activated

### ISSUE THREAD

**Issue by jasoloz** [TobyLobster/ImportLDraw#32]
Hi, I found this error making a tutorial about your add-on for Spanish users. Using Blender default language (International Fonts unchecked in System preferences) works OK, there is only a warning due to a custom colour from an old version of MLCad (this is not the problem). In this case the console shows:

```
10:24:31.81 [importldraw] The LDraw Parts Library path to be used is: c:\temp\p\
tnt\LDraw
10:24:31.81 [importldraw] Use LSynth Parts requested
10:24:31.81 [importldraw] Standard-res primitives selected
10:24:31.81 [importldraw] Loading files
10:24:32.75 [importldraw] Creating NodeGroups
10:24:32.75 [importldraw] Creating Blender objects
10:24:33.46 [importldraw] WARNING: WARNING: Could not decode 50331648 to a colou
r
10:24:34.70 [importldraw] Number of vertices: 259278
10:24:35.02 [importldraw] Number of vertices of convex hull: 231
10:24:35.02 [importldraw] Centre object
10:24:35.02 [importldraw] Adding 147 objects to scene
10:24:35.02 [importldraw] Load Done

```
But when International Fonts is checked and Spanish is selected, the import does not work:

```
10:21:58.74 [importldraw] The LDraw Parts Library path to be used is: c:\temp\p\
tnt\LDraw
10:21:58.74 [importldraw] Use LSynth Parts requested
10:21:58.74 [importldraw] Standard-res primitives selected
10:21:58.74 [importldraw] Loading files
10:22:00.13 [importldraw] Creating NodeGroups
10:22:00.13 [importldraw] createBlenderDistanceToCenterNodeGroup #create
10:22:00.13 [importldraw] createBlenderVectorElementPowerNodeGroup #create
10:22:00.13 [importldraw] createBlenderConvertToNormalsNodeGroup #create
10:22:00.13 [importldraw] createBlenderConcaveWallsNodeGroup #create
10:22:00.13 [importldraw] createBlenderSlopeTextureNodeGroup #create
10:22:00.13 [importldraw] createBlenderFresnelNodeGroup #create
10:22:00.13 [importldraw] createBlenderReflectionNodeGroup #create
10:22:00.13 [importldraw] createBlenderDielectricNodeGroup #create
10:22:00.13 [importldraw] createBlenderLegoStandardNodeGroup #create
10:22:00.13 [importldraw] createBlenderLegoTransparentNodeGroup #create
10:22:00.14 [importldraw] createBlenderLegoTransparentFluorescentNodeGroup #crea
te
10:22:00.14 [importldraw] createBlenderLegoRubberNodeGroup #create
10:22:00.14 [importldraw] createBlenderLegoRubberTranslucentNodeGroup #create
10:22:00.14 [importldraw] createBlenderLegoEmissionNodeGroup #create
10:22:00.14 [importldraw] createBlenderLegoChromeNodeGroup #create
10:22:00.14 [importldraw] createBlenderLegoPearlescentNodeGroup #create
10:22:00.14 [importldraw] createBlenderLegoMetalNodeGroup #create
10:22:00.14 [importldraw] createBlenderLegoGlitterNodeGroup #create
10:22:00.14 [importldraw] createBlenderLegoSpeckleNodeGroup #create
10:22:00.14 [importldraw] createBlenderLegoMilkyWhiteNodeGroup #create
10:22:00.14 [importldraw] Creating Blender objects
Traceback (most recent call last):
  File "C:\Users\josealfonso.LABVIRTUAL\AppData\Roaming\Blender Foundation\Blend
er\2.79\scripts\addons\io_scene_importldraw\importldraw.py", line 385, in execut
e
    loadldraw.loadFromFile(self, self.filepath)
  File "C:\Users\josealfonso.LABVIRTUAL\AppData\Roaming\Blender Foundation\Blend
er\2.79\scripts\addons\io_scene_importldraw\loadldraw\loadldraw.py", line 3954,
in loadFromFile
    rootOb = createBlenderObjectsFromNode(node, node.matrix, name)
  File "C:\Users\josealfonso.LABVIRTUAL\AppData\Roaming\Blender Foundation\Blend
er\2.79\scripts\addons\io_scene_importldraw\loadldraw\loadldraw.py", line 3446,
in createBlenderObjectsFromNode
    createBlenderObjectsFromNode(childNode, childNode.matrix, childNode.filename
, childColourName, blenderParentTransform, localToWorldSpaceMatrix * localMatrix
, blenderNodeParent)
  File "C:\Users\josealfonso.LABVIRTUAL\AppData\Roaming\Blender Foundation\Blend
er\2.79\scripts\addons\io_scene_importldraw\loadldraw\loadldraw.py", line 3311,
in createBlenderObjectsFromNode
    mesh, newMeshCreated = createMesh(name, meshName, geometry)
  File "C:\Users\josealfonso.LABVIRTUAL\AppData\Roaming\Blender Foundation\Blend
er\2.79\scripts\addons\io_scene_importldraw\loadldraw\loadldraw.py", line 3220,
in createMesh
    material = BlenderMaterials.getMaterial(faceColour, isSlopeMaterial)
  File "C:\Users\josealfonso.LABVIRTUAL\AppData\Roaming\Blender Foundation\Blend
er\2.79\scripts\addons\io_scene_importldraw\loadldraw\loadldraw.py", line 2164,
in getMaterial
    material = BlenderMaterials.__createNodeBasedMaterial(blenderName, col, isSl
opeMaterial)
  File "C:\Users\josealfonso.LABVIRTUAL\AppData\Roaming\Blender Foundation\Blend
er\2.79\scripts\addons\io_scene_importldraw\loadldraw\loadldraw.py", line 1725,
in __createNodeBasedMaterial
    BlenderMaterials.__createCyclesConcaveWalls(nodes, links, 0.2)
  File "C:\Users\josealfonso.LABVIRTUAL\AppData\Roaming\Blender Foundation\Blend
er\2.79\scripts\addons\io_scene_importldraw\loadldraw\loadldraw.py", line 2023,
in __createCyclesConcaveWalls
    out = nodes['Group']
KeyError: 'bpy_prop_collection[key]: key "Group" not found'

location: <unknown location>:-1

location: <unknown location>:-1
```

**Comment by jasoloz** [comment]
I think I have solved the problem. It appears when "New data" button is checked ("Nuevos datos" in Spanish). In Python it is UserPreferencesSystem.use_translate_new_dataname. In that case Blender creates the translated key "Grupo" instead of "Group" and throws the keyError. I can work without "New data" checked, only Interface and Tooltips options are required to work with Blender in Spanish.

**Comment by TobyLobster** [comment]
This is now fixed in the latest release (1.1.9) - thank you for reporting the bug.
